doc/ref_arch/kubernetes/chapters/chapter04.md

```diff
@@ -84,50 +84,51 @@ feature-gates:
 - container-d (with CRI plugin enabled, which it is by default)
 - Docker CE (via the dockershim, which is currently built in to the kubelet)
 - CRI-O
 - Frakti
 
-To support `req.sec.gen.04` the architecture specifies the usage of a container runtime with the capability of Kernel isolation:
+To support `req.inf.vir.01` the architecture specifies the usage of a container runtime with the capability of Kernel isolation:
 - kata-containers
 
-These specifications cover the [full lifecycle of a container](https://github.com/opencontainers/runtime-spec/blob/master/runtime.md#lifecycle) `creating > created > running > stopped` which includes the use of storage that is required during this lifecycle - this is management of the Host OS filesystem by the container runtime. This lifecycle management by the container runtime (when conformant with the above specifications) supports the requirement `req.inf.stg.06` for ephemeral storage for Pods.
+These specifications cover the [full lifecycle of a container](https://github.com/opencontainers/runtime-spec/blob/master/runtime.md#lifecycle) `creating > created > running > stopped` which includes the use of storage that is required during this lifecycle - this is management of the Host OS filesystem by the container runtime. This lifecycle management by the container runtime (when conformant with the above specifications) supports the ephemeral storage for Pods.
 
-To fulfill `req.sec.gen.05` the architecture specifies the usage of the Kubernetes CPU Manager and to support the isolation of workload resources from the infrastructure resources [CPU Pooler](https://github.com/nokia/CPU-Pooler/).
+To support the isolation of the resources used by the infrastructure from the resources used by the workloads the architecture specifies the usage of the Kubernetes CPU Manager and [CPU Pooler](https://github.com/nokia/CPU-Pooler/).
 
 > Todo: details and RA2 specifications relating to runtimes in order to meet RM features and requirements from RM chapters 4 and 5.
 
 <a name="4.5"></a>
 ## 4.5 CNI plugins
 
-> Editors note: The following chapter lists a set of CNI plugins conformant with the Reference Architecture. In future releases the list of CNI plugins should be refined in a way that there is only one example component selected for each functionality. 
-
-The selected CNI multiplexer/metapulgin may be [DANM](https://github.com/nokia/danm) as it provides the possibility to use several other CNI plugins (`req.inf.ntw.16`) and provides an API based solution to administer the networks (`req.inf.ntw.10`) from a central point (`req.inf.ntw.11`).<br>
+The selected CNI multiplexer/metapulgin may be [DANM](https://github.com/nokia/danm) as it provides the possibility to use several other CNI plugins (`req.inf.ntw.06`) and provides an API based solution to administer the networks from a central point (`req.inf.ntw.03`).<br>
```

Comment:
If the API is required by a CNI plugin, I suggest the language be updated to be more neutral:

> The selected CNI multiplexer/metapulgin should support other CNI plugins (req.inf.ntw.06) and provide an API based solution to administer the networks from a central point (req.inf.ntw.03).

Otherwise make the management API be optional for a CNI:

>  The selected CNI multiplexer/metapulgin MUST support other CNI plugins (req.inf.ntw.06) and MAY provide an API based solution to administer the networks from a central point (req.inf.ntw.03).

Reply:
Thanks I've updated the wording based on this.

Reply:
@taylor @pgoyal01 - could one/both of you review and approve the PR if you're happy (given the points made in the call about going into more detail and potentially changing the recommendation in future iterations). Thanks!